browserslist

```diff
@@ -2,6 +2,6 @@
 # https://github.com/chris-pearce/scally#browser-support
 # https://github.com/ai/browserslist
 
-last 2 versions
-Android >= 2
-Explorer >= 9
\ No newline at end of file
+ie 10
```

Comment:
Supporting ie10 has potential impacts to how we implement flexbox, we should discuss more.